lib/ansible/modules/git.py

```diff
@@ -150,7 +150,7 @@
 
     single_branch:
         description:
-            - Clone only the history leading to the tip of the specified I(branch).
+            - Clone only the history leading to the tip of the specified I(version).
```

Comment:
Rather than using a parameter name, use a generic term the indicate a valid `get` ref.
```suggestion
            - Clone only the history leading to the tip of the specified revision.
```

Reply:
Done.